pkg/riff/commands/handler.go

```diff
@@ -29,7 +29,21 @@ func NewHandlerCommand(ctx context.Context, c *cli.Config) *cobra.Command {
 		Use:   "handler",
 		Short: "handlers map HTTP requests to applications, functions or images",
 		Long: strings.TrimSpace(`
-<todo>
+Handlers can be created for one of an application, function or image.
+Application and function based handlers continuously watch for the latest built
+image and will deploy new images. If the underlying application or function is
+deleted, the handler will continue to run, but will no longer self update. Image
+based handlers must be manually updated to trigger rollout of an updated image.
+
+Applications, functions and images are logically identical at runtime. Functions
+are more focused and opinionated applications, and images are compiled
```

Comment:
Not sure about this. Functions do not handle HTTP so how can they be a specific kind of application? Are we really talking about function _images_ here?

Reply:
added `with an invoker`

Function + Invoker = Application